CHANGELOG.md

```diff
@@ -78,6 +78,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   Rustc and Xcode.
   (#[1492](https://github.com/nix-rust/nix/pull/1492))
 
+- Deprecated `SockAddr/InetAddr::to_str` in favor of `ToString::to_string`
+  (#[1447](https://github.com/nix-rust/nix/pull/1447))
```

Comment:
Looks like a copypasta in the CHANGELOG PR number.  Fix that and I'll merge the PR.